packages/tsconfig-reference/copy/id/options/importsNotUsedAsValues.md

```diff
@@ -0,0 +1,14 @@
+---
+display: "Impor Tidak Digunakan Sebagai Nilai"
+oneline: "Mengontrol sintaks yang Anda gunakan untuk mengimpor kata"
+---
+
+Kode ini mengontrol cara kerja `import`, ada 3 opsi berbeda:
+
+- `remove`: Perilaku umum menghapus pernyataan `import` yang hanya acuan kata.
+
+- `preserve`: Mempertahankan semua pernyataan `import` yang nilai atau kata tidak pernah digunakan. Hal ini dapat menyebabkan impor/efek samping yang dipertahankan.
+
+- `error`: Ini mempertahankan semua impor (sama seperti pilihan), tetapi akan mengalami galat saat impor nilai hanya digunakan sebagai kata. Ini mungkin berguna jika Anda ingin memastikan tidak ada nilai yang diimpor secara tidak sengaja, tetapi tetap membuat impor dengan jelas.
```

Comment:
nilai hanya digunakan sebagai kata, ini bahasa Inggrisnya apa?
Saya curiga ini sepertinya mengacu pada istilah teknis di TypeScript.

Reply:
`but will error when a value import is only used as a type.` ini bahasa inggris nya mas .
Bagaimana ?

Reply:
tetapi akan terjadi galat ketika nilai impor hanya digunakan sebagai tipe data.

Reply:
Kayaknya sering terjadi miskonsepsi `type` istilah teknis di TypeScript dengan type yg lain